src/pages/admin-ui-sdk/release-notes.md

```diff
@@ -8,6 +8,16 @@ keywords:
 
 # Adobe Commerce Admin UI SDK release notes
 
+## Version 1.1.2
+
+### Release date
+
+October 10, 2023
```

Comment:
We'll release tomorrow at 5:00 pm CEST (my time), should we update this to October 6?